### DISCUSSION TITLE
希望大佬可以支持其他网盘 比如夸克网盘

### DISCUSSION THREAD

**Discussion by gzxiexl** [messense/aliyundrive-webdav#305] [Ideas]
目前夸克网盘速度和容量非常可观了 
希望大佬能够考虑添加夸克网盘

**Comment by stankyoztc** [comment]
确实如此,阿里居然没有把阿里云盘和88会员绑定.反而送了夸克云盘6T.淘宝拼多多1块钱就能买夸克年费6T了.这还能说啥,真香
